spec/brainstem/api_docs/formatters/markdown/endpoint_formatter_spec.rb

```diff
@@ -184,6 +190,36 @@ module Markdown
                   it "includes the legacy status if specified" do
                     expect(subject.output).to include "Legacy: false"
                   end
+
+                  context "when required option is specified" do
```

Comment:
Any assertion we can make to ensure that this is true. Usually I go on to folks about how easy it is to end up in a different state than we think we're in (seen it so many times), but I know that you know what you're doing.

Good be as easy as

```rb
context "when required is true" do
  before do
    expect(show_config.dig(:valid_params, :sprocket_id, :required)).to eq(true)
  end
```